loopd/config.go

```diff
@@ -183,6 +185,9 @@ type Config struct {
 	TotalPaymentTimeout time.Duration `long:"totalpaymenttimeout" description:"The timeout to use for off-chain payments."`
 	MaxPaymentRetries   int           `long:"maxpaymentretries" description:"The maximum number of times an off-chain payment may be retried."`
 
+	MaxStaticAddrHtlcFeePercentage       float64 `long:"maxstaticaddrhtlcfeepercentage" description:"The maximum fee percentage that the server can charge for the htlc tx."`
+	MaxStaticAddrHtlcBackupFeePercentage float64 `long:"maxstaticaddrhtlcbackupfeepercentage" description:"The maximum fee percentage that the server can charge for the htlc backup tx."`
```

Comment:
It worth noting somewhere (in the comment) that backup transaction is either a high fee transaction or an extremely high fee transaction. I found this in `staticaddr/loopin/actions.go` by grepping over the code :-)

Reply:
good point! Added more context in this comment.